ddtrace/tracer/span.go

```diff
@@ -337,6 +338,27 @@ func (s *span) String() string {
 	return strings.Join(lines, "\n")
 }
 
+// Format implements fmt.Formatter.
+func (s *span) Format(f fmt.State, c rune) {
+	switch c {
+	case 's':
+		fmt.Fprint(f, s.String())
+	case 'v':
+		if tr, ok := internal.GetGlobalTracer().(*tracer); ok {
+			fmt.Fprintf(f, "dd.service=%s ", tr.config.serviceName)
+		}
+		if e := s.Meta[ext.Environment]; e != "" {
+			fmt.Fprintf(f, "dd.env=%s ", e)
+		}
+		if v := s.Meta[ext.Version]; v != "" {
```

Comment:
We might need to change this, since `version` won't be set for spans that aren't under the "main" service.

Can we source env/service/version all from the global tracer?

Reply:
We can't easily source env from the global tracer since it is implemented as a global tag.